lms/templates/learner_dashboard/certificate.underscore

```diff
@@ -1,7 +1,7 @@
 <div class="certificate-container">
-    <p class="title"><%- gettext('XSeries Program Certificates') %>:</p>
+    <h2 class="hd-6"><%- gettext('XSeries Program Certificates') %>:</h2>
```

Comment:
I'm not a fan of these ``hd-x`` classes. They aren't semantic, but then they also aren't very descriptive either. I'd like to see us remove them from the Pattern library. Given that, I'd prefer that you style h2s either by adding a semantic class (say ``certificate-title``) or just styling all h2s the same way within ``certificate-container``.